tests/suites/os/suite.js

```diff
@@ -23,15 +23,41 @@ module.exports = {
 		const Worker = this.require('common/worker');
 		// The balenaOS class contains information on the OS image to be flashed, and methods to configure it
 		const BalenaOS = this.require('components/os/balenaos');
+		const utils = this.require('common/utils');
+		const worker = new Worker(this.suite.deviceType.slug, this.getLogger());
```

Comment:
Isn't a new instance of something preferable to keep inline in case the function is refactored to be called more than once?

I don't think what you have is breaking, I'm just wondering why you moved this particular line.

Reply:
This file is so small it shouldn't matter... just wondering if there is a preferred syntax

Reply:
@klutchell `waitForServiceState()` requires both `utils.waitUntil` and `worker.executeCommandInHostOS`